Makefile

```diff
@@ -21,6 +21,9 @@ clean-devenv:
 devshell:
 	condev-shell || true
 
+docs:
+	$(MAKE) -C docs docs
```

Comment:
Add a `docs` target to the top-level `Makefile` so that `make docs` works from the root directory of the clone.